doc/start/documenting-ceph.rst

```diff
@@ -181,35 +203,46 @@ You must also remove any reference to a deleted document from other documents.
 Build the Source
 ----------------
 
-To build the documentation, navigate to the ``ceph`` repository directory::
+To build the documentation, navigate to the ``ceph`` repository directory:
+
+
+.. prompt:: bash $
 
 	cd ceph
 
-To build the documentation on Debian/Ubuntu, Fedora, or CentOS/RHEL, execute::
+To build the documentation on Debian/Ubuntu, Fedora, or CentOS/RHEL, execute:
+
+.. prompt:: bash $
+
+	admin/./build-doc
 
-	admin/build-doc
+To scan for the reachability of external links, execute:
 
-To scan for the reachability of external links, execute::
+.. prompt:: bash $
 
-	admin/build-doc linkcheck
+	admin/./build-doc linkcheck
```

Comment:
why?

Reply:
Because this command doesn't work (in my experience) unless it's prepended with the ./

If that's not the case elsewhere, I'll change it, but I'd like to understand what's going on first.

Reply:
could you define "doesn't work"? for instance, what the symptom was?

Reply:
By "doesn't work", I mean that the command "build-doc" doesn't work if it is run like this:

$ build-doc

However, the command "build-doc" does work if it is run like this:

$ ./build-doc

Reply:
1. you changed `admin/build-doc` to `admin/./build-doc` while you were using `build-doc`, i don't think there are any connection between these two things
1. you failed to explain the symptom.
1. please note, we are not referencing `build-doc` anywhere in our document.
1. if you insist using `build-doc`, please ensure that your `PATH` environment variable includes the directory where `build-doc` is located.

Reply:
1. I don't know what you're meaning to say.
2. This isn't what "symptom" means. However, in number 4 you've answered the question that I had, so I won't pursue it.
3. We literally say this in the documentation: "To build the documentation on Debian/Ubuntu, Fedora, or CentOS/RHEL, execute: admin/build-doc". I don't know what you mean when you say that we don't reference it anywhere in the document. It's right there.
4. Your number 4 is the answer to this issue, and is useful, and I'll be adding it to this document in a note. Unless the directory that contains ``build-doc`` is on the path, you have to prepend a dot-slash to the command. I'll direct readers to do this. As I mention it here, though, I think it's also a good idea to say publicly that this is not the way that I build documentation locally for testing. I prepend a dot-slash, and I don't have the directory on my path.

Reply:
@zdover23 all i want is to emphasis the difference between `build-doc` and `admin/build-doc` in this context. and your change does not address the issue.

Reply:
@tchaikov It looks like I don't know what the difference is between ``build-doc`` and ``admin/build-doc``. What is it?